lumicks/pylake/kymotracker/detail/calibrated_images.py

```diff
@@ -67,12 +67,18 @@ def get_rect(self, rect):
 
     @property
     def line_time_seconds(self):
-        return self.time_step / 1e9
+        return self.time_step_ns / 1e9
 
     def from_coord(self, coord):
+        """Convert from coordinates to pixels. Note that it rounds the position.
+
+        Note that it rounds the pixel position."""
```

Comment:
the `"Note that it rounds the position"` on the first line should be removed

Reply:
Good catch. Not sure why I didn't see it.